patches/@spectrum-css+component-builder+1.0.1.patch

```diff
@@ -10,18 +10,20 @@ index 0eb302e..cc7d4d3 100644
  var valueParser = require('postcss-value-parser');
  let varUtils = require('../lib/varUtils.js');
  
-@@ -40,33 +39,36 @@ async function fetchVars() {
+@@ -40,33 +39,38 @@ async function fetchVars() {
    return true;
  }
  
 -module.exports = postcss.plugin('postcss-custom-properties-mapping', function() {
 -  return async function (root, result) {
 -    await fetchVars();
++let fetchVarsPromise;
 +module.exports = function() {
 +  return {
 +    postcssPlugin: 'postcss-custom-properties-mapping',
 +    OnceExit: async function (root, result) {
-+      await fetchVars();
++      fetchVarsPromise ??= fetchVars();
++      await fetchVarsPromise;
```

Comment:
This ensures we only parse all of the spectrum variables once per build instead of for every individual file.

Reply:
can we remove this now that the parcel pr was merged?

Reply:
no, this is unrelated to parcel.